ludwig/automl/base_config.py

```diff
@@ -40,10 +40,12 @@
 
 encoder_defaults = {"text": {"bert": os.path.join(CONFIG_DIR, "text/bert_config.yaml")}}
 
-SMALL_DISTINCT_COUNT = 20
+# For a given feature, the highest percentage of distinct values out of the total number of rows that we might still
+# assign the CATEGORY type.
+CATEGORY_TYPE_DISTINCT_VALUE_PERCENTAGE_CUTOFF = 0.5
 
-# Set >= SMALL_DISTINCT_COUNT & >= MAX_DISTINCT_BOOL_PERMUTATIONS
-MAX_DISTINCT_VALUES_TO_RETURN = 100
+# Cap for number of distinct values to return.
+MAX_DISTINCT_VALUES_TO_RETURN = 10000
```

Comment:
This is much higher than the current value.  I recall @tgaddair being sensitive to this from the scaling standpoint; is this new number okay from that standpoint?

Reply:
Reverted back to the original value as discussed. As long as `FieldInfo.num_distinct_values` contains the true, non-capped number of distinct values, then we can simply use that.